src/taxes/utils/utils.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  CountryWithCodeFragment,
+  TaxClassFragment,
+  TaxRateFragment
+} from "@saleor/graphql";
+
+export const getDefaultTaxRateInCountry = (
+  taxClasses: TaxClassFragment[],
+  selectedCountry: CountryWithCodeFragment
+): TaxRateFragment["rate"] =>
+  taxClasses
+    .find(taxClass => taxClass.isDefault)
+    .countries.find(country => country.country.code === selectedCountry.code)
+    .rate;
```

Comment:
Shouldn't ts throw error here since find can theoretically return undefined?

Reply:
There is always at least one default tax class. Whole taxClasses can be undefined when query is in progress, though. I'll update the types to include undefined here